Sources/App/Core/Github.swift

```diff
@@ -249,6 +249,53 @@ extension Github {
         var htmlUrl: String
     }
 
+    struct FundingLink: Codable, Equatable {
+
+        enum Platform: String, Codable {
+            case communityBridge
+            case customUrl
+            case gitHub
+            case issueHunt
+            case koFi
+            case lfxCrowdfunding
+            case liberaPay
+            case openCollective
+            case otechie
+            case patreon
+            case tideLift
+
+            private static let gitHubApiEncodings: [String: Platform] = [
```

Comment:
This (and the associated `init(from:)`) feels a little weird, but I think it's better than having two different enums that have the same cases but different raw values, and it's certainly better than having the SCREAMING_SNAKE_CASE in our database JSON.

Reply:
Do I understand correctly that the API can send both `COMMUNITY_BRIDGE` _and_ `communityBridge` as possible values and that's why we to do this?